jobs/build-arch.Jenkinsfile

```diff
@@ -177,7 +157,7 @@ lock(resource: "build-${params.STREAM}-${params.ARCH}", extra: [[resource: "rele
         git clone --depth=1 https://github.com/coreos/fedora-coreos-releng-automation /var/tmp/fcos-releng
         """)
 
-        if (official) {
+        if (utils.pathExists("/etc/fedora-messaging-cfg/fedmsg.toml")) {
```

Comment:
this somewhat negates the simplification if we have to add back logic for if the pipeline is "official" or not.

Maybe instead of littering these official checks throughout the code we could have a single check (pipeutils function) that runs up front and does sanity checks based on if it's "official" or not. WDYT?

Reply:
> I guess the alternative is that we just keep the notion of "official" but drop just the developer prefix bits.

That sounds good to me. I think it'd make the intent clearer. I definitely agree we should be using it sparingly though.